packages/data-stores/src/queries/use-launchpad-navigator.ts

```diff
@@ -0,0 +1,44 @@
+import { useQuery } from '@tanstack/react-query';
+import apiFetch from '@wordpress/api-fetch';
+import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
+
+export interface APIFetchOptions {
+	global: boolean;
+	path: string;
+}
+
+interface LaunchpadNavigatorResponse {
+	available_checklists: string[];
+	current_checklist: string | null;
+}
+
+export const fetchLaunchpadNavigator = (
+	siteSlug: string | null
+): Promise< LaunchpadNavigatorResponse > => {
+	const slug = encodeURIComponent( siteSlug as string );
```

Comment:
What happens if `siteSlug` is `null`? It seems like `encodeURIComponent` when given `null` as an argument (which TS doesn't complain about because it's casted to `string`) will return `"null"` which will then result in a request to `/sites/null/launchpad/navigator`.

If you want to allow `null` as possible value for `fetchLaunchpadNavigator` it's better to handle that case instead of casting it to `string` 2 lines below.

Also, if `siteSlug` is a requirement for the query to run it's good practice to use the `enabled` option of `useQuery`.